src/commands/System/echo.ts

```diff
@@ -11,7 +11,7 @@ export default class extends SkyraCommand {
 			extendedHelp: language => language.tget('COMMAND_ECHO_EXTENDED'),
 			guarded: true,
 			permissionLevel: 10,
-			usage: '[channel:channel] [message:string] [...]',
+			usage: '[channel:channelname] [message:string] [...]',
```

Comment:
Nit: `channelname` can conflict with `string` here. Replace it with any of the following:

```suggestion
			usage: '[channel:channelname{2}] [message:string] [...]',
```

```suggestion
			usage: '[channel:channel] [message:string] [...]',
```